start-from-zero/Javascript/src/data/xhrData.json

```diff
@@ -0,0 +1,375 @@
+{
+    "inputPlaceholder": {
+        
+        "media": [
+            {
+                "user": "Username",
+                "class": "input-username",
+                "id": "username-id",
+                "name": "username",
+                "placeholder": "Username",
+                "type": "text"
+            },
+            {
+                "user": "password",
+                "class": "input-password",
+                "id": "password-id",
+                "name": "password",
+                "placeholder": "Password",
+                "type": "text"
+            },
+            {
+                "type": "date",
+                "class": "input-calendar",
+                "name": "calendar",
+                "id": "calendar-id",
+                "placeholder": "",
+                "type": "date"
+            }
+        ]
+    },
+
+    "technicalData" : {
+        "title": "Technical Level",
+        "headers": ["Trainee", "Junior", "Middle", "Senior"],
+        "columnData": [
+            {
+                "inputLevels": ["Trainee"]
+            },
+            {
+                "inputLevels": ["Junior1", "Junior2", "Junior3"]
+            },
+            {
+                "inputLevels": ["Middle1", "Middle2", "Middle3"]
+            },
+            {
+                "inputLevels": ["Senior1", "Senior2", "Senior3"]
+            }
+        ]
+    },
+
+    "textarea" : { 
+        "text": [
+            {
+                "label": "Should the candidate be hired?",
+                "id": "textarea1",
+                "placeholder": "The type of project that is suitable for the candidate Is guidance required for the candidate"
+            },
+            {
+                "label": "General impression",
+                "id": "textarea2",
+                "placeholder": "The type of project that is suitable for the candidate Is guidance required for the candidate"
+            },
+            {
+                "label": "Workflow, Leadership and Soft Skills",
+                "id": "textarea3",
+                "placeholder": "Describe the environment in which the candidate works. Describe any guidance or management experience."
+            }
+        ]
+    },
+
+    "fieldset" : {
+        "data": [
+            {
+                "fieldsetName": "OOP, Design Patterns",
+                "items": [
+                    {
+                        "span": "Classes",
+                        "name": "classes",
+                        "optionValue": ["0", "1", "2", "3"],

```

Comment:
These `optionValue` and `dropdown` properties seem to be repeated all over the place. Can we do anything to try to keep them a bit more DRY?